edx-platform/pearson-vue-theme/lms/templates/header/user_dropdown.html

```diff
@@ -43,17 +42,23 @@
         % if resume_block:
             <div class="mobile-nav-item dropdown-item dropdown-nav-item"><a href="${resume_block}" role="menuitem">${_("Resume your last course")}</a></div>
         % endif
-        % if not enterprise_customer_portal:
-            <div class="mobile-nav-item dropdown-item dropdown-nav-item"><a href="${reverse('dashboard')}" role="menuitem">${_("Dashboard")}</a></div>
-        % else:
-            <div class="mobile-nav-item dropdown-item dropdown-nav-item"><a href="${settings.ENTERPRISE_LEARNER_PORTAL_BASE_URL}/${enterprise_customer_portal.get('slug')}" role="menuitem">${_("Dashboard")}</a></div>
+        % if configuration_helpers.get_value('SHOW_DASHBOARD_IN_DROPDOWN', False):
+            <div class="mobile-nav-item dropdown-item dropdown-nav-item">
+                <a href="${reverse('dashboard')}" role="menuitem">${_("Dashboard")}</a>
+            </div>
+        % endif
+        % if configuration_helpers.get_value('SHOW_PROFILE_IN_DROPDOWN', False):
+            <div class="mobile-nav-item dropdown-item dropdown-nav-item">
+                <a href="${reverse('learner_profile', kwargs={'username': username})}" role="menuitem">${_("Profile")}</a>
+            </div>
         % endif
-
-        <div class="mobile-nav-item dropdown-item dropdown-nav-item"><a href="${reverse('learner_profile', kwargs={'username': username})}" role="menuitem">${_("Profile")}</a></div>
         <div class="mobile-nav-item dropdown-item dropdown-nav-item"><a href="${reverse('account_settings')}" role="menuitem">${_("Account")}</a></div>
-        % if should_show_order_history:
+        % if should_redirect_to_order_history_microfrontend():
             <div class="mobile-nav-item dropdown-item dropdown-nav-item"><a href="${settings.ORDER_HISTORY_MICROFRONTEND_URL}" role="menuitem">${_("Order History")}</a></div>
         % endif
-        <div class="mobile-nav-item dropdown-item dropdown-nav-item"><a href="${reverse('logout')}" role="menuitem">${_("Sign Out")}</a></div>
+        % if configuration_helpers.get_value('EXTERNAL_SUPPORT_LINK', ''):
```

Comment:
Are we using this setting in any of the prod sites? @sergivalero20

Reply:
@Squirrel18 yes, we're using it in several sites.